static/data/community.ts

```diff
@@ -15,6 +15,16 @@ const communityChat = {
   subtitle:
     'The Podman developers are generally around during CEST and Eastern Time business hours, so please be patient if you’re in another time zone!',
   links: [
+    {
+      text: 'Slack - Podman Community',
+      path: 'https://app.slack.com/huddle/T08PSQ7BQ/C08MXJLCFCN',
+      icon: 'logos:slack-icon',
+    },
+    {
+      text: 'Slack - Podman Developers',
```

Comment:
I agree with Tom the two slack links next to each other look odd and it is not clear to a normal person what the difference would be. Maybe it would be better to not list the dev channel here and only list the -dev channel as part of the channel description or something as most